bag_replicator_unpack_to_archive/src/main/scala/uk/ac/wellcome/platform/archive/bagreplicator/unpack_to_archive/services/UnpackedBagService.scala

```diff
@@ -0,0 +1,31 @@
+package uk.ac.wellcome.platform.archive.bagreplicator.unpack_to_archive.services
+
+import java.io.InputStream
+
+import com.amazonaws.services.s3.AmazonS3
+import uk.ac.wellcome.platform.archive.common.ConvertibleToInputStream._
+import uk.ac.wellcome.platform.archive.common.bagit.{BagInfoLocator, S3BagFile}
+import uk.ac.wellcome.platform.archive.common.models.bagit.{BagInfo, ExternalIdentifier}
+import uk.ac.wellcome.platform.archive.common.parsers.BagInfoParser
+import uk.ac.wellcome.storage.ObjectLocation
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class UnpackedBagService(s3Client: AmazonS3)(implicit ec: ExecutionContext) {
```

Comment:
I don't think this needs to share the idea of an unpacked bag. This is kind of a `BagLocator` maybe?